Monopoly/Assets/Scripts/Game.cs

```diff
@@ -295,4 +388,25 @@ void SetJail()
         currentPlayer.GoToJail();
         infoPopup.ShowMessage("Idziesz do więzienia", "Będziesz pauzował 3 tury");
     }
-}
+
+    void HandlePlayerBancrupcy(Player debtor, Player creditor)
+    {
+        foreach(Property property in debtor.ownedProperties)
+        {
+            creditor.ownedProperties.Add(property);
+            property.ChangeOwner(creditor.GetId());
```

Comment:
Mam wrażenie że mamy źle zrobione to przypisywanie property graczom. Teraz trzeba zrobić dwie rzeczy osobno:
- wpisać property na listę ownedProperties u gracza,
- zmienić coś wewnątrz property

mogłoby to być zebrane w jakiejś jednej metodzie? np. metodzie klasy Property, do której pakowałoby się jako argument obiekt Player